ablySpec/Crypto.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  Crypto.swift
+//  ably
+//
+//  Created by Toni Cárdenas on 12/2/16.
+//  Copyright © 2016 Ably. All rights reserved.
+//
+
+import Nimble
+import Quick
+import SwiftyJSON
+
+class Crypto : QuickSpec {
+    override func spec() {
+        describe("Crypto") {
+            for keyLength in ["128", "256"] {
```

Comment:
Did something similar: https://github.com/ably/ably-ios/pull/209/files#diff-ef1119dd0f908b6d236fe852b7c431f7R152 for #209